src/i18n/swg-strings.ts

```diff
@@ -512,45 +512,44 @@ export const SWG_I18N_STRINGS = {
     'pt': 'Obrigado por ver este anúncio',
     'pt-br': 'Agradecemos por assistir este anúncio',
     'ru': 'Спасибо, что посмотрели эту рекламу!',
-    'se': 'Tack för att du tittade på den här annonsen',
+    'sv': 'Tack för att du tittade på den här annonsen',
     'th': 'ขอบคุณที่ดูโฆษณานี้',
     'tr': 'Bu reklamı görüntülediğiniz için teşekkür ederiz',
     'uk': 'Дякуємо, що переглянули це оголошення',
     'zh-cn': '感谢观看此广告',
     'zh-hk': '感謝觀看此廣告',
     'zh-tw': '感謝觀看這則廣告',
   },
-  // TODO: mhkawano - Update once TC translations are done
   'VIEW_AN_AD': {
     'en': 'View an ad',
-    'ar': 'View an ad',
-    'de': 'View an ad',
+    'ar': 'عرض إعلان',
+    'de': 'Anzeige ansehen',
     'en-au': 'View an ad',
     'en-ca': 'View an ad',
     'en-gb': 'View an ad',
     'en-us': 'View an ad',
-    'es': 'View an ad',
-    'es-419': 'View an ad',
-    'fr': 'View an ad',
-    'fr-ca': 'View an ad',
-    'hi': 'View an ad',
-    'id': 'View an ad',
-    'it': 'View an ad',
-    'ja': 'View an ad',
-    'ko': 'View an ad',
-    'ms': 'View an ad',
-    'nl': 'View an ad',
-    'no': 'View an ad',
-    'pl': 'View an ad',
-    'pt': 'View an ad',
-    'pt-br': 'Veja um anúncio',
-    'ru': 'View an ad',
-    'se': 'View an ad',
-    'th': 'View an ad',
-    'tr': 'View an ad',
-    'uk': 'View an ad',
-    'zh-cn': 'View an ad',
-    'zh-hk': 'View an ad',
-    'zh-tw': 'View an ad',
+    'es': 'Ver un anuncio',
+    'es-419': 'Ver un anuncio',
+    'fr': 'Visionner une annonce',
+    'fr-ca': 'Afficher une annonce',
+    'hi': 'कोई विज्ञापन देखें',
+    'id': 'Lihat iklan',
+    'it': 'Visualizza un annuncio',
+    'ja': '広告を表示',
+    'ko': 'V광고 보기',
```

Comment:
Looks like 'V' is a typo here? (As a korean I can tell this is wrong to be there)